plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/MinishiftLocationSection.java

```diff
@@ -136,7 +136,7 @@ public void widgetDefaultSelected(SelectionEvent e) {
 		msHomeModListener = new ModifyListener() {
 			@Override
 			public void modifyText(ModifyEvent e) {
-				execute(new SetLocationPropertyCommand(server));
+				execute(new SetMinishiftHomePropertyCommand(server));
```

Comment:
This fixes JBIDE-24796.   Sorry for the confusion, I was using this branch as a collector until master is open.